osu.Game/Graphics/UserInterface/Nub.cs

```diff
@@ -163,14 +165,20 @@ public Color4 GlowColour
 
         private void onCurrentValueChanged(ValueChangedEvent<bool> filled)
         {
-            fill.FadeTo(filled.NewValue ? 1 : 0, 200, Easing.OutQuint);
+            const double duration = 200;
+
+            fill.FadeTo(filled.NewValue ? 1 : 0, duration, Easing.OutQuint);
 
             if (filled.NewValue)
-                main.ResizeWidthTo(1, animate_in_duration, Easing.OutElasticHalf);
+            {
+                main.ResizeWidthTo(1, duration, Easing.OutElasticHalf);
+                main.TransformTo(nameof(BorderThickness), filled.NewValue ? 8.5f : border_width, duration, Easing.OutElasticHalf);
```

Comment:
Bit weird to be checking `filled.NewValue` again inside an `if` branch that already ensures it to be true.

```suggestion
                main.TransformTo(nameof(BorderThickness), 8.5f, duration, Easing.OutElasticHalf);
```